app/assets/stylesheets/ltags/GithubTag.scss

```diff
@@ -337,23 +330,63 @@
 .gh-btn-container {
   text-align: center;
   padding: 0.35em 0 1.35em;
-  box-shadow: -0px -0px 60px 42px #fff;
-  box-shadow: -0px -0px 60px 42px var(--card-bg);
+  box-shadow: -0 -0 60px 42px var(--card-bg);
   position: relative;
   z-index: 100;
-}
 
-.gh-btn {
-  color: #0366d6;
-  background-color: #f1f8ff;
-  border-radius: 3px;
-  line-height: 20px;
-  padding: 0.25em 1.2em;
-  opacity: 0.9;
-  font-size: 0.94em;
-  font-weight: bold;
-  border: 1px solid #0366d6;
-  &:hover {
-    opacity: 1;
+  .gh-btn {
```

Comment:
(refactor/blocking) - just a small thing but to keep this in step with the crayons button, we could use some scss syntax here 🙂 

In your class selector here, add

```scss
    @extend .crayons-btn;
    @extend .crayons-btn--secondary;
```

and for this to work, at the top of the file you will need to add another import:

`@import '../components/buttons';`

This will save you needing to copy all these over, but give the same effect 👍 

Let me know if any trouble with this! I've tested locally and all is looking great with both old and new posts - nice work!

Reply:
@aitchiss cool that works! thank you